android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -523,6 +536,21 @@ Call.Issue getIssueFromString(String issue) {
     }
     return Call.Issue.NOT_REPORTED;
   }
+
+  CallMessage.MessageType getMessageTypeFromString (String messageType) {
+    if (messageType.equals(CommonConstants.UserDefinedMessage)) {
+      return CallMessage.MessageType.UserDefinedMessage;
+    }
+    return CallMessage.MessageType.UserDefinedMessage;
+  }
+
+  String getContentTypeFromString (String contentType) {
+    if (contentType.equals(CommonConstants.ApplicationJson)) {
```

Comment:
same as above

Reply:
done made private 👍 

@mhuynh5757 I see the method above this `getIssueFromString` defaults return value, so I followed similar pattern. Any opinions on throwing an error to user?

Reply:
Hmm, let's see. I can see it being useful, but a bit more work in the long run. These functions don't really do validation, more like type guarding. We already ensure that the passed values are enumeration members in the JS layer, so it's not strictly necessary to throw here, but it would reduce errors like typos on our end down the road. I think we should do it, but it will also create a bit more work when we have new enumeration members. When we support more types, we'll have to modify both the native and JS layer. But that seems OK to me, what do you think?

Reply:
I have added the error to Java layer. It can't hurt to add more validation where we can.

Reply:
Very nice, let's do one more thing though. If we look at the control flow here, we're returning `null` to the builder which will probably cause the Java runtime to crash. For example, let's say message type is invalid and we return `null`. The Java runtime is still executing, and passes `null` to the builder, which will cause an exception. So we're going to want to do these validations earlier, the promise will be rejected properly, and then we want to return instead of throwing.